### DISCUSSION TITLE
一个第三方下载链接

### DISCUSSION THREAD

**Discussion by Makitoid** [ClassIsland/ClassIsland#415] [常规]
我搞了个onedrive链接[戳这里](https://hm6ds-my.sharepoint.com/:f:/g/personal/ban_hm6ds_onmicrosoft_com/Eup1xZ7vr2xJq77GGV2WB1MBXrxFKbKL6Ue_6fYAWbKbCA?e=UeMbMs)
密码是ClassIsland
支持任何人上传，可编辑

**Comment by Makitoid** [comment]
谁能把它加到文档里？

**Comment by jiangyin14** [comment]
支持任何人上传的操作是并不安全的，这可能导致计算机病毒以 ClassIsland 为载体传播。
故请修改OneDrive共享策略，否则此社区源大概率不会被加入Docs中。

**Comment by Makitoid** [comment]
> 故请修改OneDrive共享策略，否则此社区源大概率不会被加入Docs中。

已更改
